cryptocurrency-sentiment-analysis/src/main/java/com/hazelcast/jet/demo/CryptocurrencySentimentAnalysis.java

```diff
@@ -106,10 +112,20 @@ private static Pipeline buildPipeline() {
                                             .flatMap(ct -> ct.markers().stream())
                                             .collect(toList());
         StreamStage<String> tweets = pipeline
-                .readFrom(twitterSource(allCoinMarkers))
+                .readFrom(TwitterSources.timestampedStream(
+                        loadCredentials(),
+                        () -> new StatusesFilterEndpoint()
+                                .trackTerms(allCoinMarkers)))
                 .withNativeTimestamps(SECONDS.toMillis(1));
-
         StreamStageWithKey<Entry<CoinType, Double>, CoinType> tweetsWithSentiment = tweets
+                .map(rawTweet -> {
+                    try {
+                        return new JSONObject(rawTweet).getString("text");
```

Comment:
Could it possible that Twitter returns a response without the `text` field ?

Reply:
It normally returns the response with the `text` field. But in abnormal situations, I do not want to stop the job. The old version of this demo also had this kind of verification in the deleted `TwitterSource` file. In order to increase readability, I can remove this try-catch clause if you want.

Reply:
This is for demonstration purposes only so I favor readability to please remove that